packages/theme/css/global.css

```diff
@@ -5,16 +5,15 @@
  * It would be nice to do these as a global reset, so each component doesn't need to set these.
  * - font-family
  * - line-height
- * - background
  * - color
  */
 
-p {
+ .uitk-theme {
```

Comment:
@joshwooding this came up in a 101, but would be good to further discuss this or get your opinion